### ISSUE TITLE
Small request 

### ISSUE THREAD

**Issue by donavanbecker** [colinbendell/homebridge-blink-for-home#50]
Can you hide or only enable during debug mode, the large amount of logs that are displaying the json coming from the blink api?

**Comment by colinbendell** [comment]
Turn off debug and verbose mode should remove that (it’s under advanced
settings)

**Comment by donavanbecker** [comment]
```

{
            "name": "Blink",
            "username": "email",
            "password": "password",
            "pin": pin,
            "hide-alarm": false,
            "hide-manual-arm-switch": false,
            "hide-privacy-switch": false,
            "avoid-thumbnail-battery-drain": true,
            "enable-verbose-logging": false,
            "enable-debug-logging": false,
            "enable-startup-diagnostic": false,
            "_bridge": {
                "username": "0E:F4:A7:91:49:97",
                "port": 35030
            },
            "platform": "Blink"
        }

```

**Comment by xImportExportx** [comment]
i am having the same issue @colinbendell, seems to be ingnoring the verbose false flag.

**Comment by mtressl** [comment]
Same here.

**Comment by mtressl** [comment]
More context: when verbose logging is enabled, the following call puts a huge response in the log.

[2/21/2021, 12:43:23 AM] [Blink Security] GET /api/v3/accounts/xxxxx/homescreen

**Comment by rweckhoff** [comment]
Same here ... all are false, however, significant logging of API calls/data:

       {
            "name": "Blink",
            "username": "redacted",
            "password": "redacted",
            "pin": 123456,
            "enable-liveview": false,
            "avoid-thumbnail-battery-drain": true,
            "camera-thumbnail-refresh-seconds": 3600,
            "enable-verbose-logging": false,
            "enable-debug-logging": false,
            "enable-startup-diagnostic": false,
            "platform": "Blink"
        },

**Comment by donavanbecker** [comment]
fixed with v3.7.1
